src/components/authInput/index.ts

```diff
@@ -0,0 +1,25 @@
+import template from "./template.hbs";
+import Component from "../../utils/Component"
+import * as styles from "./styles.module.scss";
+
+class Input extends Component {
+  props: {
+    errorText: string,
+    name: string,
+    type: string,
+    value: string,
+    label: string,
+    styles: {
+      input:string
+    }
+  };
```

Comment:
Если ты хотел описать тип пропсов, то лучше это сделать через ключевое слово type или interface. Сейчас описание пропсов является свойством класса, что не совсем корректно. Свойством могут быть уже конкретный объект пропсов.
```suggestion
  type TProps = {
    errorText: string,
    name: string,
    type: string,
    value: string,
    label: string,
    styles: {
      input:string
    }
  };
```